content/quickstart/aws/modify-program.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Modify the Program
+weight: 7
+menu:
+  quickstart:
+    parent: aws
+    identifier: aws-modify-program
+---
+
+Now that we have an instance of our Pulumi program deployed, let's update it to do something a little more interesting.
+
+Replace the entire contents of {{< langfile >}} with the following:
+
+{{< langchoose nogo >}}
+
+```javascript
+"use strict";
+const pulumi = require("@pulumi/pulumi");
+const aws = require("@pulumi/aws");
+const awsx = require("@pulumi/awsx");
+
+const size = "t2.micro"; // t2.micro is available in the AWS free tier
+
+const ami = "ami-6869aa05" // AMI for us-east-1 (Virginia)
```

Comment:
I wish we never used this example. It means there's a high likelihood the example won't work if someone selects a different AWS region during the configuration step. TBH, I'd like to kill this example with fire in our examples repo.

A better approach would be to fetch the AMI dynamically:

```typescript
// Look up the right AMI for running Ubuntu Trusty in our region.
const ami = aws.getAmi({
    filters: [
        { name: "name", values: ["ubuntu/images/hvm-ssd/ubuntu-trusty-14.04-amd64-server-*"] },
        { name: "virtualization-type", values: ["hvm"] },
    ],
    owners: ["099720109477"], // Canonical
});
```

Reply:
Fixed for JS/TS, but couldn't get Python working so leaving Python as-is for now, with an extra note about updating the `ami` variable. Will address subsequently.